util/src/main/java/tech/pegasys/artemis/util/config/LoggingDestination.java

```diff
@@ -11,7 +11,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.logging;
+package tech.pegasys.artemis.util.config;
```

Comment:
Ultimately,`util` is the wrong place for logging-specific constants.  But fixing this issue will require a larger refactor.  Will discuss offline when / how to prioritize refactoring.

Reply:
sounds good!